### PR TITLE
Add path alias field to page edit form

### PR DESCRIPTION
**Pull request by ekes**
Required for https://github.com/localgovdrupal/localgov_microsites/issues/146
Also seems standard configuration for LocalGov Drupal content types.

The second commit https://github.com/localgovdrupal/localgov_page/commit/3f1cfcf2528ad2f3ffa9d1edbf793ded3f089df2 is separated but was just in the exported config.

### CODE REVIEW

File: config/install/core.entity_form_display.node.localgov_page.default.yml [hunk 1]
```diff
@@ -32,13 +32,19 @@ content:
       require_layouts: 1
     third_party_settings: {  }
   localgov_page_summary:
-    type: text_textarea
+    type: string_textarea
```
Comment (finnlewis) [inline config/install/core.entity_form_display.node.localgov_page.default.yml]: @ekes just checking this is an intentional change.

**Review by stephen-cox — APPROVED** [review]
Working and looking fine for me.

File: config/install/core.entity_form_display.node.localgov_page.default.yml (hunk 1 above)
Comment (stephen-cox) [reply config/install/core.entity_form_display.node.localgov_page.default.yml]: This change was committed to the 1.x branch with https://github.com/localgovdrupal/localgov_page/commit/84ed080af8bfcee098e3d16ea302a414f73cb239, so should disappear with the resolution of the merge conflict.

**Comment by finnlewis** [comment]
@ekes needs a tweak - conflicts

**Comment by finnlewis** [comment]
Closing this as superseded

**Pull request closed without merging**